modules/andes-core/client/src/main/java/org/wso2/andes/client/AMQConnectionDelegate_8_0.java

```diff
@@ -116,6 +116,11 @@ public ProtocolVersion makeBrokerConnection(BrokerDetails brokerDetail) throws A
             settings.setTrustStoreCertType(sslConfig.getCertType());
         }
 
+        if(settings.getKeyStorePath() == null && settings.getKeyStorePassword() ==null && settings
+                .getTrustStorePath() == null && settings.getTrustStorePassword() == null){
+              settings.loadSSLConfigFromSysConfig();
```

Comment:
Shall we move this block to above "if( sslConfig != null)" block and add a log to say we are loading from system config?